docs/linting.md

```diff
@@ -92,6 +92,17 @@ By default, we filter out generated files from linting.
 
 To bypass this filter, add `tags=["lint-genfiles"]` to a target to force all the `srcs` to be linted.
 
+## Ignoring files explicitly
```

Comment:
this is the wrong file, I think you want to edit `formatting.md`. This one has nothing to do with `format.sh`

Reply:
Done